### DISCUSSION TITLE
When can BPMN be supported？

### DISCUSSION THREAD

**Discussion by zoujiaqing** [elsa-workflows/elsa-core#2557] [General]
BPMN is a very general workflow specification. I don't know when version 3.0 can be experienced.

**Comment by sfmskywalker** [comment]
It's hard to predict when 3.0 will be available, but my current guess is early Q2 of 2022. But that will not include BPMN. Although there is the desire to add support for BPMN, I'm afraid I have no idea when someone may be able to work on it.

**Reply by sjd2021** [reply]
Looks like it's the next checkbox in the roadmap for 3.0. Do you know if any headway has been made? No rush; I just wanted to respond to someone who'd recently asked about BPMN support.
